app/views/notes/_form.html.erb

```diff
@@ -0,0 +1,42 @@
+<div><h3><%= form_title %></h3></div>
+
+<div>
+  <% if note.errors.any? %>
+    <% note.errors.full_messages.each do |msg| %>
+      <p> <%= msg %></p>
+    <% end %>
+  <% end %>
+</div>
```

Comment:
We already have flash messages in the application.html.erb, this can be removed and the error rendered there